src/Microsoft.DotNet.XHarness.Android/AdbRunner.cs

```diff
@@ -487,7 +487,7 @@ public void PushFiles(string localDirectory, string devicePath, bool removeIfPre
                 return null;
             }
 
-            if (allDevicesAndTheirProperties.Any(kvp => kvp.Value.Equals(apkRequiredProperty, StringComparison.OrdinalIgnoreCase)))
+            if (allDevicesAndTheirProperties.Any(kvp => kvp.Value.Equals(apkRequiredProperty, StringComparison.OrdinalIgnoreCase) == true))

```

Comment:
If I remember correctly, there was a `?` somewhere behind `kvp` or `Value`. Can you please double check the merge?

Reply:
Nice catch, thanks! Added it